lib/howl/interactions/buffer_selection.moon

```diff
@@ -58,11 +58,17 @@ get_buffer_list = ->
   for basename, buffers in pairs(basenames)
     continue if #buffers == 1
 
-    titles = [make_title(buffer, project:true) for buffer in *buffers]
-    if has_duplicates(titles)
-      titles = [make_title(buffer, project:true, parents:1) for buffer in *buffers]
-      if has_duplicates(titles)
-        titles = [make_title(buffer, project:true, parents:2) for buffer in *buffers]
+    options_list = {
+      { project: false, parents: 1 }
+      { project: false, parents: 2 }
+      { project: true, parents: 1 }
```

Comment:
There should also be just a {project: true} somewhere in here? i.e. just the project name with no parents.